templates/common/__init__.py.in

```diff
@@ -183,22 +183,29 @@ def Analyze(command, sa_clang_compiler, sa_formatting=False, sa_modernize="", sa
     Perform static analysis and check of formatting rules if requested
     '''
     returnVal=0
+    formatting_on = os.environ.has_key("SA_FORMATTING") or sa_formatting
+    modernize_on = os.environ.has_key("SA_MODERNIZE") or sa_modernize
```

Comment:
Also, please change the `or sa_modernize` (and `sa_formatting`) to check against an empty string `or sa_modernize != ""`